source/compatibility.tex

```diff
@@ -2708,7 +2708,7 @@
 \tcode{sizeof(char*)}
 in C.
 \difficulty
-Programs must add explicit casts to the appropriate rvalue.
+Programs are required to add explicit casts to the appropriate rvalue.
```

Comment:
This is one of the "synonyms" for "shall" that we can't use, either (see https://www.iso.org/sites/directives/current/part2/index.xhtml#_idTextAnchor091).